examples/nnp_training_force.py

```diff
@@ -33,12 +34,7 @@
 ShfA = torch.tensor([9.0000000e-01, 1.5500000e+00, 2.2000000e+00, 2.8500000e+00], device=device)
 num_species = 4
 aev_computer = torchani.AEVComputer(Rcr, Rca, EtaR, ShfR, EtaA, Zeta, ShfA, ShfZ, num_species)
-energy_shifter = torchani.utils.EnergyShifter([
-    -0.600952980000,  # H
-    -38.08316124000,  # C
-    -54.70775770000,  # N
-    -75.19446356000,  # O
-])
+energy_shifter = torchani.utils.EnergyShifter([None])
```

Comment:
I think it should be `EnergyShifter(None)`

Reply:
fixed in https://github.com/aiqm/torchani/pull/283/commits/2125d7fc80a8abf822c89c3f0e510ab4eea3abc2